examples/deployment/kubernetes/deploy.sh

```diff
@@ -3,50 +3,72 @@
 go get github.com/google/trillian/...
 cd $GOPATH/src/github.com/google/trillian
 
-# Build docker images
-docker build -f examples/deployment/docker/log_server/Dockerfile -t gcr.io/$PROJECT_NAME/log_server:$TAG .
-docker build -f examples/deployment/docker/log_signer/Dockerfile -t gcr.io/$PROJECT_NAME/log_signer:$TAG .
+echo "Building docker images..."
+docker build --quiet -f examples/deployment/docker/log_server/Dockerfile -t gcr.io/$PROJECT_NAME/log_server:$IMAGE_TAG .
+docker build --quiet -f examples/deployment/docker/log_signer/Dockerfile -t gcr.io/$PROJECT_NAME/log_signer:$IMAGE_TAG .
 # TODO(al): when cloudspanner supports maps:
 # docker build -f examples/deployment/docker/map_server/Dockerfile -t us.gcr.io/$PROJECT_NAME/map_server:$TAG .
 
-# Connect to gcloud
-gcloud config set project "${PROJECT_NAME}"
-gcloud config set compute/zone "${ZONE}"
-
-# Configure Docker to use gcloud credentials with Google Container Registry
-gcloud auth configure-docker
+echo "Pushing docker images..."
+gcloud docker -- push gcr.io/${PROJECT_NAME}/log_server:${IMAGE_TAG}
+gcloud docker -- push gcr.io/${PROJECT_NAME}/log_signer:${IMAGE_TAG}
 
-# Push docker images
-docker push "gcr.io/${PROJECT_NAME}/log_server:${TAG}"
-docker push "gcr.io/${PROJECT_NAME}/log_signer:${TAG}"
+echo "Tagging docker images..."
+gcloud --quiet container images add-tag gcr.io/${PROJECT_NAME}/log_server:${IMAGE_TAG} gcr.io/${PROJECT_NAME}/log_server:latest
+gcloud --quiet container images add-tag gcr.io/${PROJECT_NAME}/log_signer:${IMAGE_TAG} gcr.io/${PROJECT_NAME}/log_signer:latest
 
 # TODO(al): when cloudspanner supports maps:
 # gcloud docker -- push "us.gcr.io/${PROJECT_NAME}/map_server:${TAG}"
 
+echo "Updating jobs..."
 # Prepare configmap:
-kubectl delete configmap deploy-config
-envsubst < examples/deployment/kubernetes/trillian-cloudspanner.yaml | kubectl create -f -
+kubectl delete configmap deploy-config || true
+envsubst < ${CONFIGMAP} | kubectl create -f -
 
 # Launch with kubernetes
-envsubst < examples/deployment/kubernetes/trillian-log-deployment.yaml | kubectl apply -f -
-envsubst < examples/deployment/kubernetes/trillian-log-service.yaml | kubectl apply -f -
-envsubst < examples/deployment/kubernetes/trillian-log-signer-deployment.yaml | kubectl apply -f -
-envsubst < examples/deployment/kubernetes/trillian-log-signer-service.yaml | kubectl apply -f -
-kubectl get all
-kubectl get services
+envsubst < ${DIR}/trillian-log-deployment.yaml | kubectl apply -f -
+envsubst < ${DIR}/trillian-log-service.yaml | kubectl apply -f -
+envsubst < ${DIR}/trillian-log-signer-deployment.yaml | kubectl apply -f -
+envsubst < ${DIR}/trillian-log-signer-service.yaml | kubectl apply -f -
+kubectl set image deployment/trillian-logserver-deployment trillian-logserver=gcr.io/${PROJECT_NAME}/log_server:${IMAGE_TAG}
+kubectl set image deployment/trillian-logsigner-deployment trillian-log-signer=gcr.io/${PROJECT_NAME}/log_signer:${IMAGE_TAG}
 
 # TODO(al): Create trees
```

Comment:
nit: Could either delete this TODO or replace it with `./provision_tree.sh` (and change README.md appropriately).

Reply:
Deleted - I think it makes more sense to leave it as a separate step.